model/src/main/kotlin/PackageCurationData.kt

```diff
@@ -174,6 +183,7 @@ data class PackageCurationData(
             purl = purl ?: other.purl,
             cpe = cpe ?: other.cpe,
             authors = (authors.orEmpty() + other.authors.orEmpty()).toSortedSet(),
+            copyrightHolders = (copyrightHolders.orEmpty() + other.copyrightHolders.orEmpty()).toSortedSet(),
```

Comment:
I believe this change is ok to do in this commit, as the commit is about splitting out `copyrightHolders` from `authors` while in generate I believe that it is wrong: the semantics of the curation is to overwrite any existing copyright holders with the given one. This function deviates from the overwrite semantics. 

Do you agree and if so, shall we add a FIXME comment to fix the issue for both, `authors` and `copyrightHolders`?

Reply:
The intention is to override any copyright finding or authorship  with the values from curation.
The code, as it stands, is merging both authors and copyright findings which contradicts the intened behavior.

IMHO you are actually raising a very good point here that validates a FIXME comment.

Reply:
Done